src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -938,7 +939,7 @@ case class WorkflowActor(workflow: WorkflowDescriptor, backend: Backend)
         self ! CallFailed(collector, None, e.getMessage)
       case Success(outputs) =>
         logger.info(s"Collection complete for Scattered Call ${collector.tag}.")
-        self ! CallCompleted(collector, outputs, 0)
+        self ! CallCompleted(collector, outputs, Seq.empty, 0)
```

Comment:
Just a warning... I think all of the changes in this file will conflict pretty heavily with the `job_avoidance` branch.